src/sas/qtgui/Plotting/Slicers/BoxSlicer.py

```diff
@@ -236,9 +289,11 @@ def getParams(self):
 
         """
         params = {}
-        params["x_max"] = numpy.fabs(self.vertical_lines.x)
-        params["y_max"] = numpy.fabs(self.horizontal_lines.y)
+        params["x_width"] = self.xwidth
+        params["y_width"] = self.ywidth
```

Comment:
The other `params` keys match their respective class dictionary keys. If this were true for `(x|y)width`, this could allow for much more straightforward (de)serialization of the slicer objects.